website/project/model.py

```diff
@@ -529,6 +524,8 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     #: Whether this is a pointer or not
     primary = True
 
+    is_draft_registration = False
```

Comment:
In general, I'm not thrilled with the number of similar-sounding flags we're accumulating on the `Node` model. Is there any way to do this in a more concise fashion? I'd be curious to sit down and learn a bit more about the design constraints for having all these flags on the same class, while other registration info- eg draft registration- stays a separate model instance.

We currently have the following members that seem relevant to registrations, all on `Node`:
- `is_pending_registration`
- `is_registration_approved`
- `is_retracted`
- `is_pending_registration`
- `is_pending_embargo`
` is_pending_embargo_for_existing_registration`
- `is_registration`
- `is_registration_of`
- `is_derived_from`